sct.py

```diff
@@ -561,16 +562,33 @@ def conf_docs(output_format):
 @cli.command("perf-regression-report", help="Generate and send performance regression report")
 @click.option("-i", "--es-id", required=True, type=str, help="Id of the run in Elastic Search")
 @click.option("-e", "--emails", required=True, type=str, help="Comma separated list of emails. Example a@b.com,c@d.com")
-def perf_regression_report(es_id, emails):
+@click.option("-n", "--with-nemesis", required=False, type=bool, default=False, is_flag=True,
```

Comment:
may be better to rename --emails -> --emails-recipients, so it will be same as for send_email function